src/Nox.Types.Abstractions/Enums/NoxType.cs

```diff
@@ -128,10 +128,10 @@ public enum NoxType : uint
     [SimpleType(typeof(string))]
     DateTimeSchedule = 3271762546,
 
-    [SimpleType(typeof(byte))]
+    [SimpleType(typeof(int))]
```

Comment:
To be aligned with Value object
```suggestion
    [SimpleType(typeof(ushort))]
```

Reply:
update from main and regenerate the migrations

Reply:
done